docs/agent_container_manual_deploy.md

```diff
@@ -47,17 +47,17 @@ If the management hub you are using uses Secure Socket Layer (SSL) encryption, t
 2. Prior to starting the container, export the following variables:
 
    * HZN_AGENT_IMAGE=\<name of the agent container image in the repo\>
-     * This should include the url for the repository if it is not the default one.
+      * This should include the url for the repository if it is not the default one.
```

Comment:
The indentation of the sub-bullets is correct.   The alignment should be directly under the first character of parent bullet.  See the correct rendering at https://open-horizon.github.io/docs/anax/docs/agent_container_manual_deploy/

Same comment for the below edits.  The doc is already correct.